gh-124476: Fix decoding from the locale encoding in the C.UTF-8 locale (#132477)
python/cpython: Misc/NEWS.d/next/Core_and_Builtins/2025-04-13-17-18-01.gh-issue-124476.fvGfQ7.rst

> Fix decoding from the locale encoding on C.UTF-8 locale.

Reviewer: ```suggestion
Fix decoding of locale encoding in the C.UTF-8 locale.
```

Reviewer 2: I prefer to say "decoding from" (and "encode to").

Reviewer: ```suggestion
Fix decoding from the locale encoding in the C.UTF-8 locale.
```
Is this better?
